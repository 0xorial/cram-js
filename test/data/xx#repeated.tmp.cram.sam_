@SQ	SN:xx	LN:20	M5:bbf4de6d8497a119dda6e074521643dc	UR:/Users/rbuels/dev/htslib/test/xx.fa
S	67	xx	1	1	10M	=	11	20	AAAAAAAAAA	**********	MD:Z:10	NM:i:0
S	131	xx	11	1	10M	=	1	-20	TTTTTTTTTT	**********	MD:Z:10	NM:i:0
S	67	xx	1	1	10M	=	11	20	AAAAAAAAAA	**********	MD:Z:10	NM:i:0
S	131	xx	11	1	10M	=	1	-20	TTTTTTTTTT	**********	MD:Z:10	NM:i:0
S	67	xx	1	1	10M	=	11	20	AAAAAAAAAA	**********	MD:Z:10	NM:i:0
S	131	xx	11	1	10M	=	1	-20	TTTTTTTTTT	**********	MD:Z:10	NM:i:0
