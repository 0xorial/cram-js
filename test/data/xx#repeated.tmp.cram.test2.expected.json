[
  {
    "tags": {},
    "flags": 67,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 1,
    "readGroupId": -1,
    "readName": "S",
    "recordsToNextFragment": 1,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 0
  },
  {
    "tags": {},
    "flags": 131,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 11,
    "readGroupId": -1,
    "readName": "S",
    "recordsToNextFragment": 2,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 1
  },
  {
    "tags": {},
    "flags": 67,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 1,
    "readGroupId": -1,
    "readName": "S",
    "recordsToNextFragment": 3,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 2
  },
  {
    "tags": {},
    "flags": 131,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 11,
    "readGroupId": -1,
    "readName": "S",
    "recordsToNextFragment": 4,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 3
  },
  {
    "tags": {},
    "flags": 67,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 1,
    "readGroupId": -1,
    "readName": "S",
    "recordsToNextFragment": 5,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 4
  },
  {
    "tags": {},
    "flags": 131,
    "cramFlags": 1,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 11,
    "readGroupId": -1,
    "readName": "S",
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 5
  }
]